src/rt/util/typeinfo.d

```diff
@@ -247,7 +247,8 @@ unittest
         {
             assert(f1 == 0 + 0i);
 
-            assert(f1  == f2);
+            assert(f1 == f2);
+            version (DragonFlyBSD) {} else   // FIXME:  //core.exception.AssertError@src/rt/util/typeinfo.d(249): unittest failure
             assert(f1 !is f2);
```

Comment:
This is the failing assertion.

I think i might need some help from one of the 'src/rt/typeinfo/*' code owners on how to debug this.

(The {} are not missing, it's only trying to exclude a single line, without having to change it's surroundings, Wanted to keep the 'DragonFlyBSD' string and 'FIXME' on a single line, so it's easy to find using grep. I noticed this 'unittest exclusion' pattern in other commits.).

Reply:
@mathias-lang-sociomantic @Geod24 @WalterBright @redstar
Can one of you help me figure out why/how 'assert(f1 !is f2);' could be failing.
Could it be that comparing complex numbers is causing trouble. Does one of you have a good way of tracing ?